packages/calcite-components/src/components/block/block.stories.ts

```diff
@@ -275,3 +275,36 @@ export const icons_TestOnly = (): string => html`
     </calcite-block-section>
   </calcite-block>
 `;
+
+export const actionsEndLoadingIconStartEnd = (): string => html`
+  <calcite-block
+    heading="Valid status"
+    description="summary"
+    collapsible
+    icon-start="pen"
+    icon-end="pen"
+    style="width: 500px"
+  >
+    <calcite-icon
+      icon="compass"
+      slot="content-start"
+      style="color: var(--calcite-color-status-success)"
+      scale="s"
+    ></calcite-icon>
+    <calcite-action appearance="transparent" icon="ellipsis" text="menu" label="menu" slot="actions-end" />
+  </calcite-block>
+`;
+
+export const actionsEndContentStartIconStartEnd = (): string => html`
```

Comment:
If both screenshot tests are adding coverage for start/end icons, can you merge them into a single test and add info for each test case (similar to [this](https://github.com/Esri/calcite-design-system/blob/99eac2e4e63f96c95eb354883921b3f56bf7841a/packages/calcite-components/src/components/slider/slider.stories.ts#L466-L524))?